.pylintrc

```diff
@@ -3,7 +3,7 @@
 # A comma-separated list of package or module names from where C extensions may
 # be loaded. Extensions are loading into the active Python interpreter and may
 # run arbitrary code
-extension-pkg-whitelist=
+extension-pkg-whitelist=lxml.etree
```

Comment:
What's this for?

Reply:
weird intersection of `feedgen` and typeshed's `lxml.etree` implementation. vendor compatibility

Reply:
Can you document this in the ``.pylintrc``? Good to preserve context for unusual things